### ISSUE TITLE
hammr template build fails with "unknown error in template file" if builder section is missing

### ISSUE THREAD

**Issue by alefebvr** [usharesoft/hammr#224]
When running the command:

`hammr template build --file template.yml`

if file template.yml only has the stack section and not the builder section, then hammr returns:
```
you provided a yaml file, checking...
ERROR: unknown error in template file
```
Instead, a more explicit error message should be returned (such as "file has no builder section").

**Comment by bastienmartin** [comment]
Tested with hammr 3.8.4, this issue is not fixed.

**Comment by AliceAlbano** [comment]
Expected behavior: clearer error message like "missing builder section in file FILENAME"

**Comment by taorian** [comment]
fixed by #437
